en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-configuration.md

```diff
@@ -0,0 +1,90 @@
+  
+  3. Select **Storage** section and click **S3**.
+  
+     <img src="/assets/img/connectors/amazon-s3-create-bucket-sample-1.png" title="Select S3 services" width="800" alt="Select S3 services"/>
+
+  4. Create a bucket.
+  
+     <img src="/assets/img/connectors/amazon-s3-create-bucket-sample-2.png" title="Create S3 bucket" width="800" alt="Create S3 bucket"/>
+       
+## Create Deployment Package
+
+  Your function's code consists of scripts or compiled programs and their dependencies. When you author functions in the Lambda console or a toolkit, the client creates a ZIP archive of your code called a [ deployment package ](https://docs.aws.amazon.com/lambda/latest/dg/deployment-package-v2.html). The client then sends the package to the Lambda service. When you manage functions with the Lambda API, command line tools, or SDKs, you create the deployment package. You also need to create a deployment package manually for compiled languages and to add dependencies to your function.
+  
+  This sample explains hot to create a sample python program as deployment package.
+   
+  1. Create sample python function (Ex: lambda_function.py) file. (On Linux and macOS, use your preferred shell and package manager. On Windows 10, you can [install the Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to get a Windows-integrated version of Ubuntu and Bash.)
+  2. Create a ZIP archive.
+     ```
+     ~/my-function$ zip function.zip lambda_function.py
+       
+     adding: lambda_function.py (deflated 17%)
+     ```
+  3. Upload created ZIP archive into the created S3 bucket.
+   
+     <img src="/assets/img/connectors/upload-deployement-file.png" title="Upload deployment package" width="800" alt="Upload deployment package"/>
+   
+## Create Execution Role
+    
+You can use AWS Identity and Access Management (IAM) to manage access to the Lambda API and resources like functions and layers. For users and applications in your account that use Lambda, you manage permissions in a permissions policy that you can apply to IAM users, groups, or roles. To grant permissions to other accounts or AWS services that use your Lambda resources, you use a policy that applies to the resource itself.
+   
+Creating an [Execution Role](https://docs.aws.amazon.com/lambda/latest/dg/lambda-permissions.html#lambda-intro-execution-role) in the IAM Console.
+
+   1. Open the [roles page](https://console.aws.amazon.com/iam/home#/roles) in the IAM console.
+      
+      <img src="/assets/img/connectors/create-iam-roles.png" title="Create IAM roles" width="800" alt="Create IAM roles"/>
+      
+   2. Choose Create role.
+   3. Under Common use cases, choose Lambda.
+   4. Choose Next: Permissions.
+   5. Under Attach permissions policies, choose the AWSLambdaBasicExecutionRole and AWSXrayWriteOnlyAccess managed policies.
+   6. Choose Next: Tags.
+   7. Choose Next: Review.
+   8. For Role name, enter lambda-role.
+   7. Choose Create role.
```

Comment:
We need to specify what tokens to save following these steps - which will be useful to configure the connector.

Reply:
Done.